gulpfile.js

```diff
@@ -301,10 +303,36 @@ gulp.task('apps', ['dist', 'clean-apps'], function (done) {
                 process.exit(1);
             });
         }
-        done();
+        runSequence('post-build', function() {
+            done();
+        });        
     });
 });
 
+gulp.task('post-build', function (done) {
+
+    var platforms = getPlatforms();
+
+    var merged = mergeStream();
+
+    if (platforms.indexOf('linux32') != -1) {
+        // Copy Ubuntu launcher scripts to destination dir
+        var launcherDir = path.join(appsDir, pkg.name, 'linux32');
+        console.log('Copy Ubuntu launcher scripts to ' + launcherDir);
+        merged.add(gulp.src('assets/linux/**')
+            .pipe(gulp.dest(launcherDir)));
+    }
+
+    if (platforms.indexOf('linux64') != -1) {
+        // Copy Ubuntu launcher scripts to destination dir
+        var launcherDir = path.join(appsDir, pkg.name, 'linux64');        
+        console.log('Copy Ubuntu launcher scripts to ' + launcherDir);        
+        merged.add(gulp.src('assets/linux/**')
+            .pipe(gulp.dest(launcherDir)));
+    }
+
+    return merged.isEmpty() ? null : merged;
```

Comment:
You must return `done()` here when true, I found out that the tasks for platform `osx` are not correctly finished otherwise.

Reply:
I taked this line from the `gulp-merge` doc. But the call to the callback will be surely better, so `done()` wil be a better solution (if added the callback to the function).